### ISSUE TITLE
Do not shows and byes (both players lose) - Feature request

### ISSUE THREAD

**Issue by pmontrasio** [teijo/jquery-bracket#15]
In some tournaments it happens that one or both players do not show or that a player gets a bye because there is a odd number of participants or because of tournament rules (example: the top seeded players skip the first N rounds). There are many fringe cases but the most basic one is the do not show. Implementing that would be enough for implementing many tournament use cases. 

Let's consider these matches in a larger tournament.

    A ____
    B     |
          |_____ E _____
          |      F      |
    C ____|             |_____ G
    D                   |        H
                     ___|

If the player at A doesn't play, the player at B wins and advances to seat E. We could assign a default score to the game A-B, maybe 0-1. The script handles that right now without any problem.

However if both players don't play, they both lose and I believe there is no way to handle this case right now (maybe a score of 0-0?). Let's suppose this is implemented. The seat E must be marked as "permanently empty" because of the double do not show in A-B.

Let's say C-D ends with a 2-1 score. The player at C advances to seat F. The seat E is empty so that player gets a bye and advances to G in the next round.

Nothing changes if C-D ends with a do not show for any of the two player. We handle that with a default 1-0 score.

However if C-D also ends with a double do not show and both players lose, both the seats E and F are empty and that match ends with another both players lose. Nobody advances to seat G to play with whoever made it to seat H. The player at H gets a bye.

To recap, it seems that two new game results are needed: 1) "both player lose", which yields a bye in the next round and 2) "win by a bye".

Is there any chance to get that implemented? Thanks (by the way, the script is great).

**Comment by teijo** [comment]
Finally implemented and published a few days ago. Check out 0.8.0.
